armi/reactor/converters/blockConverters.py

```diff
@@ -16,17 +16,15 @@
 import copy
 import math
 
-import numpy as np
 import matplotlib
 import matplotlib.pyplot as plt
-from matplotlib.patches import Wedge
+import numpy as np
 from matplotlib.collections import PatchCollection
+from matplotlib.patches import Wedge
 
-from armi.reactor import blocks
-from armi.reactor import grids
-from armi.reactor import components
-from armi.reactor.flags import Flags
 from armi import runLog
+from armi.reactor import blocks, components, grids
```

Comment:
I see you keep doing this.  And, fine.

Just to be clear, I don't actually think this improve readability.  And I have explicitly done the opposite in hundreds of ARMI files.

Reply:
It's being done automatically by my linter, maybe I don't have it configured correctly?

Reply:
huh